Python/pythonrun.c

```diff
@@ -953,6 +954,14 @@ print_exception(PyObject *f, PyObject *value)
     if (err < 0) {
         PyErr_Clear();
     }
+    PyObject* suggestions = _Py_Offer_Suggestions(value);
+    if (suggestions) {
```

Comment:
Would you mind to add a comment showing an example of the expected output?

I understood that it's something like: `. Did you mean: (...)?`.